src/pages/index.js

```diff
@@ -122,36 +120,57 @@ cardFormValidator.enableValidation();
 
 //FORM SUBMIT
 function handleFormSubmit(data) {
-  api.profileUpdate(data).then(() => {
-    userInfo.setUserInfo(data);
-    profileUpdateSubmit.setSubmitText(true, "Saving...");
-    editProfile.close();
-  });
+  editProfile.setSubmitText(true, "Saving...");
+  api
+    .profileUpdate(data)
+    .then(() => {
+      userInfo.setUserInfo(data);
+      editProfile.setSubmitText(false);
```

Comment:
`      editProfile.setSubmitText(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.


**Please, correct it everywhere**